### PR TITLE
Add selected calendar to url

### PR DESCRIPTION
**Pull request by WadePeterson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,12 +8,12 @@
     "build": "rm -rf dist && webpack -p --config ./config/webpack.config.prod.js --progress --bail",
     "deploy:prod": "npm run build && firebase deploy",
     "lint": "eslint src test",
-    "prod": "npm run build && node server.js",
```
Comment (WadePeterson) [inline package.json]: This was also only for heroku.

File: src/app/containers/calendarContainer.js [hunk 2]
```diff
@@ -47,4 +48,15 @@ const mapPropsToRefs = (props) => {
   return [`/recipes/${props.currentUserId}`, `/calendars/${props.currentUserId}`].concat(scheduledRecipeRefs);
 };
 
-export default connect(mapStateToProps, mapDispatchToProps, mergeProps)(FirebaseUtils.firebaseConnect(mapPropsToRefs)(Calendar));
+const initialize = (props) => {
```
Comment (WadePeterson) [inline src/app/containers/calendarContainer.js]: This actually forces the `selectedCalendarId` to be set to a value if it's empty (and we have at least one calendar loaded)

File: src/app/reducers/calendar_reducer.js [hunk 3]
```diff
@@ -7,17 +8,20 @@ const initialState = {
...
-  case CHANGE_SELECTED_CALENDAR_ID:
-    return {
-      ...state,
-      selectedCalendarId: action.payload,
+    case CHANGE_CALENDAR_VIEW:
+      return {
+        ...state,
+        refDate: action.payload,
+      };
+    case LOCATION_CHANGE: {
```
Comment (WadePeterson) [inline src/app/reducers/calendar_reducer.js]: When the url changes, check if we're trying to set a new calendar id. I'm not 100% sure if I like this, but it seems like a pretty bulletproof way of doing things without race conditions and ugly code in multiple spots

**Comment by mfernandez-rally** [comment]
⛏ 🍰

**Pull request merged**